xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -544,6 +548,71 @@ rebuild_master:
         project: home:Admin
         package: ctris</screen>
         </sect4>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.set_flags">
+          <title>Set Flags for Projects, Packages, Repositories or Architectures</title>
+
+          <para>Providing the flag <emphasis>build</emphasis>, the status <emphasis>enable</emphasis> and the project <emphasis>home:Admin</emphasis> will enable <emphasis>all</emphasis> builds:</para>
```

Comment:
I think we should mention that you don't have to do *anything* because there are app-wide defaults.